lib/bibliothecary/version.rb

```diff
@@ -1,3 +1,3 @@
 module Bibliothecary
-  VERSION = "8.6.5"
+  VERSION = "8.7.0"
```

Comment:
this change doesn't break compatibility, but it could mean fewer dependencies from maven projects, so I'm considering this a bugfix / change in behavior, hence the minor version bump.